### PR TITLE
[WIP] Improved node rendering

### PR DESCRIPTION
**Pull request by corranwebster**
Better choices for connection points; use Bezier curves for lines; better arrowhead styling.

### CODE REVIEW

File: graphcanvas/graph_container.py [hunk 1]
```diff
@@ -75,16 +93,17 @@ def _apply_graphviz_layout(layout):
...
             _apply_graphviz_layout(layout)
         else:
-            layout = networkx.spring_layout(self.graph)
+            layout = networkx.spring_layout(self.graph, pos=initial_positions, scale=scale)
 
             # resize the bounds to fit the graph
             radius = numpy.log2(len(layout))
             self.bounds = [max(75, self.components[0].width)*2*radius,
                            max(50, self.components[0].height)*2*radius]
+            print self.bounds
```
Comment (stevenjkern) [inline graphcanvas/graph_container.py]: Leftover debugging print statement?

**Comment by stevenjkern** [comment]
Other than my one comment, this LG(reat)TM. It needs a `master` merge and the conflicts with #3 to be resolved, but is then good to merge.

Side note, this also fixed a bug where if a node that starts a directional edge is outside both the x and y bounds, the edge will render oddly, as below:
edge

**Comment by stevenjkern** [comment]
@corranwebster, I merged in `master`, did a little cleanup, and added some tests. If you are good with the changes, I think we can merge this.

**Comment by corranwebster** [comment]
Thanks.

**Pull request merged**